### PR TITLE
Feature/#6 as a business user i should be able to register myself and create an account

### PR DESCRIPTION
**Pull request by cs-cbonh002**

### CODE REVIEW

File: .vscode/launch.json
Comment (themellowpea) [inline .vscode/launch.json:1]: Could we untrack this folder por favor?  :)

File: rev-connect-api/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		<url/>
 	</scm>
 	<properties>
-		<java.version>17</java.version>
+		<java.version>21</java.version>
```
Comment (themellowpea) [inline rev-connect-api/pom.xml]: we should probably discuss changing java versions as a group

File: node_modules/.package-lock.json
Comment (themellowpea) [inline node_modules/.package-lock.json:1]: this is not a node project. this should be removed

**Comment by themellowpea** [comment]
It seems this PR was resolved with PR #20. We can probably rebase from `dev` and write some more tests, otherwise we can probably just close this PR

File: .vscode/launch.json
Comment (cs-cbonh002) [reply .vscode/launch.json:1]: @sarangi123 I don't use vscode so would you please add this folder to the gitignore.

File: rev-connect-api/pom.xml (hunk 1 above)
Comment (cs-cbonh002) [reply rev-connect-api/pom.xml]: It looks like this was done from @sarangi123 first commit, but I'm not sure why it was changed.

File: node_modules/.package-lock.json
Comment (cs-cbonh002) [reply node_modules/.package-lock.json:1]: @sarangi123 This looks like it was added during one of your last commits while working on the passwords. If this is not necessary to that function would you please remove it.

File: .vscode/launch.json
Comment (sarangi123) [reply .vscode/launch.json:1]: Yes. I will look into this.

**Review by themellowpea — APPROVED** [review]
Tests pass! 👍🏽

File: src/test/java/com/rev_connect_api/UserApiTests.java
Comment (revature-ben-m) [inline src/test/java/com/rev_connect_api/UserApiTests.java:1]: Should we just delete this?

**Review by revature-ben-m — APPROVED** [review]

**Pull request merged**